test/package.test.ts

```diff
@@ -38,9 +38,41 @@ describe('/package/:name/:version endpoint', () => {
     expect(json.name).toEqual(packageName);
     expect(json.version).toEqual(packageVersion);
     expect(json.dependencies).toEqual({
-      'loose-envify': '1.4.0',
-      'object-assign': '4.1.1',
-      'prop-types': '15.7.2',
+      'loose-envify': {
```

Comment:
1. We are only expanding an existing test. For such functionality I would like to see more unit tests.
2. maybe testing one of the `tricky` packages mentioned int the issue:
```
    express
    npm
    trucolor
    @snyk/snyk-docker-plugin
```
3. Are there any SLAs and did we consider any load test for an endpoint?